test/integration/auth-session-start-save.int.c

```diff
@@ -43,7 +43,7 @@ test_invoke (TSS2_SYS_CONTEXT *sapi_context)
 {
     TSS2_RC               rc;
     TPMI_SH_AUTH_SESSION  session_handle = 0;
-    TPMS_CONTEXT          context = { 0, };
+    TPMS_CONTEXT          context = { 0, 0, 0, { 0, 0 } };
```

Comment:
Given that this has showed up 4 times now I think it deserves an init macro.

Reply:
I'll add an init macro, although it really should be in the header file (which is a separate repository).